common/src/main/java/com/redhat/cloud/notifications/models/BehaviorGroup.java

```diff
@@ -43,13 +44,16 @@
  * See https://in.relation.to/2016/08/04/introducing-distinct-pass-through-query-hint/ for more details about that hint.
  */
 // TODO Move this query back to BehaviorGroupResources when hints are implemented in Mutiny.Query.
-@NamedQuery(
-        name = "findByBundleId",
-        query = "SELECT DISTINCT b FROM BehaviorGroup b LEFT JOIN FETCH b.actions a " +
-                "WHERE (b.accountId = :accountId OR b.accountId IS NULL) AND b.bundle.id = :bundleId " +
-                "ORDER BY b.created DESC, a.position ASC",
-        hints = @QueryHint(name = BehaviorGroup.HINT_PASS_DISTINCT_THROUGH, value = "false")
-)
+@NamedQueries({
+        @NamedQuery(
+                name = "findByBundleId",
+                query = "SELECT DISTINCT b FROM BehaviorGroup b LEFT JOIN FETCH b.actions a " +
+                        "WHERE (b.accountId = :accountId OR b.accountId IS NULL) AND b.bundle.id = :bundleId " +
+                        "ORDER BY b.created DESC, a.position ASC",
+                hints = @QueryHint(name = BehaviorGroup.HINT_PASS_DISTINCT_THROUGH, value = "false")),
+        @NamedQuery(name = "findByDisplayNameAndAccountId",
+                query = "SELECT b FROM BehaviorGroup b WHERE b.displayName = :displayName AND b.accountId = :accountId"),
```

Comment:
We don't need the `BehaviorGroup` data so this could be replaced with a `COUNT(*)` query.

Reply:
good catch. thanks.